.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -23,7 +23,8 @@ body:
 - type: dropdown
   attributes:
     label: Core Library
-    options: 
+    options:
+      - "MSAL.js v3-alpha (@azure/msal-browser@alpha)" 
       - "MSAL.js v2 (@azure/msal-browser)"
```

Comment:
```suggestion
      - "MSAL.js (@azure/msal-browser)"
```

Reply:
Please also make this change in all templates

Reply:
I thought I did, sorry about that.

Reply:
Didn't push the changes apparently, here they go...